includes/gateways/stripe/includes/class-give-stripe-webhooks.php

```diff
@@ -63,16 +63,23 @@ public function listen() {
 				return;
 			}
 
+			// Retrieve the request's body and parse it as JSON.
+			$body  = @file_get_contents( 'php://input' );
+			$event = json_decode( $body );
+
+			/**
+			 * Never remove this as it ensure the check for which Stripe account to use.
+			 * @Todo Utilize idempotency key on every Stripe request for better stability.
+			 */
+			$metadata    = (array) $event->data->object->metadata;
+			$donation_id = ! empty( $metadata['Donation Post ID'] ) ? $metadata['Donation Post ID'] : 0;
```

Comment:
Missing spaces before/after `'`.

Reply:
@bradp I think the static array keys don't require spaces before/after `'`. Can you please confirm the same?